common/network-common/src/test/java/org/apache/spark/network/sasl/SparkSaslSuite.java

```diff
@@ -255,21 +250,16 @@ public void testEncryptedMessageChunking() throws Exception {
 
   @Test
   public void testFileRegionEncryption() throws Exception {
-    final Map<String, String> testConf = ImmutableMap.of(
+    Map<String, String> testConf = ImmutableMap.of(
```

Comment:
Anyone wondering about 'final' -- these are no longer needed because lambdas are OK with variables that are effectively final in the enclosing scope.